File: Python-Basics/conditional_statements/godzilla_vs_kong.py
budget = float(input())
extras = int(input())
costume_price = float(input())

decor = 0.1 * budget
costumes = 0

if extras <= 150:
    costumes = extras * costume_price
elif extras > 150:
    costumes = (extras * costume_price) * 0.9

total_cost = decor + costumes

if total_cost <= budget:
    print(f'Action!')
    print(f'Wingard starts filming with {budget-total_cost:.2f} leva left.')
elif total_cost > budget:
    print(f'Not enough money!')
    print(f'Wingard needs {total_cost - budget:.2f} leva more.')

File: CPP-Basics/Godzilla vs. Kong.cpp
#include <iostream>

using namespace std;

int main () {

    double budget, clothes;
    int stuntmen;

    cin >> budget >> stuntmen >> clothes;

    double decor = 0.1 * budget;

    if (stuntmen > 150) {

        clothes = clothes * 0.9;
    }

    double cost = decor + clothes * stuntmen;
    double left, needed;

    cout.setf(ios::fixed);
    cout.precision(2);

    if (cost <= budget) {

        left = budget - cost;

    cout << "Action!" << endl << "Wingard starts filming with " << left << " leva left." << endl;

    } else {

        needed = cost - budget;

    cout << "Not enough money!" << endl << "Wingard needs " << needed << " leva more." << endl;
    }

    return 0;
}
